packages/create-start/src/cli.ts

```diff
@@ -0,0 +1,104 @@
+import { Command, createOption } from '@commander-js/extra-typings'
+import { packageManagerOption } from './modules/packageManager'
+import { logo } from './logo'
+import {
+  scaffoldTemplate,
+  templateCliOption,
+  templatePrompt,
+} from './templates'
+import { getAbsolutePath, newProjectDirectoryPrompt } from './directory'
+import { packageNameCliOption } from './modules/packageJson'
+import { ideCliOption } from './modules/ide'
+import type { TEMPLATE_NAME } from './templates'
+
+const options = {
+  template: templateCliOption,
+  packageNameCliOption: packageNameCliOption,
+  packageManager: packageManagerOption,
+  directory: createOption('--directory <string>', 'The directory to use'),
+  installDeps: createOption(
+    '--install-deps',
+    'Install dependencies after scaffolding',
+  ),
+  noInstallDeps: createOption(
+    '--no-install-deps',
+    'Skip installing dependencies after scaffolding',
+  ),
+  initGit: createOption('--init-git', 'Initialise git'),
+  noInitGit: createOption('--no-init-git', 'Skip initialising git'),
+  hideLogo: createOption('--hide-logo', 'Hide the Tanstack Start logo'),
+  ide: ideCliOption,
+}
+
+const addNewProjectOptions = (command: Command) => {
+  return command
+    .addOption(options.template)
+    .addOption(options.packageNameCliOption)
+    .addOption(options.packageManager)
+    .addOption(options.directory)
+    .addOption(options.installDeps)
+    .addOption(options.noInstallDeps)
+    .addOption(options.initGit)
+    .addOption(options.noInitGit)
+    .addOption(options.hideLogo)
+    .addOption(options.ide)
+}
+
+// const addQueryCommand = addBaseOptions(
```

Comment:
remove this?

Reply:
It will be uncommented soon. I started the add modules to test the UX but then commented them as there are no modules to add yet